src/lib/analytics/amplitude.ts

```diff
@@ -19,6 +21,8 @@ export const identify = (property: AnalyticsUserProperty) => {
   const customEvent = customIdentifyEvent({
     detail: { property: propertyTypeToLog, propertyValue: property.payload },
   });
+  dd.setContextProperty(propertyTypeToLog, property.payload);
```

Comment:
right now i just shoved the dd.setContextProperty call inside of the amplitude `identify` event since i figure there's no reason to not have them tightly coupled.

but if people are interested in an `analytics/index.ts` that has a combined method of the `identify` and `setContextProperty` functions i can do that too (or whatever other permutation of more cleanly separating these 2 that people desire)

Reply:
I think we can just create a event listener for `dydx:identify` in the datadog setup.

that we we can keep keep `src/lib/analytics` as still agnostic, and dd will simply consume the events being emitted here.